### DISCUSSION TITLE
Control / Updates via MQTT?

### DISCUSSION THREAD

**Discussion by latetedemelon** [berrysauce/hoom#7] [Q&A]
While not possible at this stage, in theory could you use this project to pair with HomeKit accessories and expose updates and controls to those accessories via MQTT?

**Comment by berrysauce** [comment]
> While not possible at this stage, in theory could you use this project to pair with HomeKit accessories and expose updates and controls to those accessories via MQTT?

Surely! I wanted to build Hoom to make HomeKit more accessible, so this sounds like an awesome idea!

**Reply by latetedemelon** [reply]
I think the scope would be the following, I'm working on trying to roll out a "stand on the sholders of others" project that attempts to send IOT info to mqtt for easier integration between devices.  I can stop back in once I've got it up and running to see if I could help?

- MQTT Broker Integration
- Define MQTT Topics and Payloads:
- Modify Hoom to Publish and Subscribe to MQTT Topics:
- Homie Convention Compliance (Optional)

https://github.com/sbtinstruments/aiomqtt

This project also appears to have tackled this issue but it appears stale:

https://github.com/forty2/mqtt2homekit
